jwst/exp_to_source/exp_to_source.py

```diff
@@ -40,8 +40,16 @@ def exp_to_source(inputs):
             log.debug(f'Copying source {slit.source_id}')
             result_slit = result[str(slit.source_id)]
             result_slit.exposures.append(slit)
+            # store values for later use (after merge_tree)
+            slit_bunit = slit.meta.bunit_data
+            slit_bunit_err = slit.meta.bunit_err
+            
```

Comment:
I think this approach could also be used to preserve the slit metadata for `slit.meta.model_type` and `slit.meta.wcsinfo`, as these should not be overridden by the exposure metadata. Otherwise seems reasonable to me.

Reply:
Ok I will add those to the list